graylog2-server/src/test/java/org/graylog/plugins/views/migrations/V20200204122000_MigrateUntypedViewsToDashboardsTest.java

```diff
@@ -73,7 +74,8 @@ public class V20200204122000_MigrateUntypedViewsToDashboardsTest {
     @Mock
     private ClusterConfigService clusterConfigService;
 
-    private final ObjectMapper objectMapper = new ObjectMapperProvider().get();
+    private final ObjectMapper objectMapper = new ObjectMapperProvider().get()
+            .setDateFormat(new StdDateFormat().withColonInTimeZone(false));
```

Comment:
Not needed.

Reply:
I must have been testing this before I put the setting in ObjectMapperProvider.
Removed.